package.json

```diff
@@ -36,6 +36,7 @@
     "lodash": "4.17.21",
     "monaco-editor": "0.33.0",
     "monaco-editor-webpack-plugin": "7.0.1",
+    "node-fetch": "^3.3.1",
```

Comment:
Strapi is using v2.6.9 of node-fetch, v3 introduces a breaking change [Converted to ES module](https://github.com/node-fetch/node-fetch/blob/main/docs/v3-UPGRADE-GUIDE.md#converted-to-es-module) that causes an error using require()
It can be fixed changing the version to "2.6.9"

Reply:
You are absolutely right. I just corrected it.
Thank you very much!